evalml/automl/automl_search.py

```diff
@@ -68,6 +68,7 @@ class AutoMLSearch:
     """Automated Pipeline search."""
     _MAX_NAME_LEN = 40
     _LARGE_DATA_ROW_THRESHOLD = int(1e5)
+    _LARGE_DATA_PERCENT_VALIDATION = 0.75
```

Comment:
@christopherbunn so, 25% training, 75% validation? Sure, that seems like a fine starting point, and certainly better than 75% training which is what we were doing before. Let's do it.